docs/source/markdown/podman-stats.1.md.in

```diff
@@ -62,16 +62,13 @@ Valid placeholders for the Go template are listed below:
 
 [1] Cgroups V1 only
 
-When using a GO template, you may precede the format with `table` to print headers.
+When using a GO template, precede the format with `table` to print headers.
```

Comment:
Not-your-fault again, but the ALL-CAPS GO looks weird to my eye. Is that a standard convention, or should it be just *Go*? (Also present in a few other places)